.nvmrc

```diff
@@ -0,0 +1 @@
+v16
```

Comment:
This change seems out of scope of this pull request

Reply:
Np, this can be removed. I was trying to enforce consistency but this should not be an issue for most people using Node 14>=

Reply:
Removed :)